src/Blocks/components/form/assets/state/init.js

```diff
@@ -547,14 +551,28 @@ export function setStateFormInitial(formId) {
 
 		const field = formElement.querySelector(`${getState([StateEnum.SELECTORS_FIELD], StateEnum.SELECTORS)}[${getStateAttribute('fieldName')}="${name}"]`);
 
-		
 		if (type ==='radio' || type ==='checkbox') {
 			setStateConditionalTagsItems(item.parentNode.parentNode.getAttribute(getStateAttribute('conditionalTags')), name, value, formId);
 		}
 
 		// Conditional tags.
 		if (field) {
-			setStateConditionalTags(field, name, formId);
+			setStateConditionalTags(field, name, false, formId);
+		}
+	}
+
+	const customFields = formElement?.querySelectorAll(getState([StateEnum.SELECTORS_FIELD_NO_FORMS_BLOCK], StateEnum.SELECTORS)) ?? {};
```

Comment:
Why is an empty object a fallback for `querySelectorAll`? A more reasonable alternative would be an empty array.

Reply:
fixed